corehq/warehouse/models/facts.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+    USER_DIM_SLUG,
+    FORM_STAGING_SLUG,
+    SYNCLOG_STAGING_SLUG,
+)
+
+from .dimensions import UserDim
+from .shared import WarehouseTableMixin
+
+
+class ApplicationStatusFact(models.Model, WarehouseTableMixin):
+    '''
+    Application Status Report Fact Table
+
+    Grain: app_id, user_id
+    '''
+    slug = APP_STATUS_FACT_SLUG
+
+    # app_dim = models.CharField(max_length=255)
+
+    # TODO is CASCADE the functionality that we want?
```

Comment:
good question. i don't think it will actually matter since we won't be using orm to populate the dims and facts probably, but i think PROTECT might be better to prevent accidental deletion of data